app/services/idv/steps/ssn_step.rb

```diff
@@ -28,6 +29,17 @@ def form_submit
         Idv::SsnFormatForm.new(current_user).submit(permit(:ssn))
       end
 
+      def invalid_state_response
```

Comment:
Little bit of noise here because I was looking at the two `ssn_step.rb` files side-by-side and wanted methods sorted etc. to ease comparison.